### PR TITLE
Add very basic monitoring and restarting of the web-connectivity test…

### PR DESCRIPTION
**Pull request by hellais**
… helper service

This is an easier version of: https://github.com/ooni/backend/issues/274

### CODE REVIEW

File: ansible/roles/th-web-connectivity/templates/if-nok-restart.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+if curl -m 5 -s {{ web_connectivity_ipv4 }}:{{ web_connectivity_port }}/status | grep '{"status": "ok"}' > /dev/null
```
Comment (bassosimone) [inline ansible/roles/th-web-connectivity/templates/if-nok-restart.sh]: do we want to add `-f` such that non `200` status codes are interpreted as bad?

Comment (hellais) [reply ansible/roles/th-web-connectivity/templates/if-nok-restart.sh]: I suppose we could do that, however the thing that matters the most is the fact that the response code matches the grep parameter.

Moreover the only case we care about in reality is the one in which curl times out, which is what happens when it enters the failed OOM state.

Comment (bassosimone) [reply ansible/roles/th-web-connectivity/templates/if-nok-restart.sh]: ok

**Comment by hellais** [comment]
Thanks for the code @FedericoCeratto !

**Pull request merged**